docs_src/conf.py

```diff
@@ -59,7 +59,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
Is this an auto-generated-docs change or did you make it?

Reply:
I did this to avoid some docs build errors.